code/modules/spacepods/pod_fab.dm

```diff
@@ -0,0 +1,488 @@
+/obj/machinery/spod_part_fabricator
```

Comment:
something makes me want to ask you why this is just a gigantic copypaste of mechfabs.